### PR TITLE
 VBM-15: add localization

### PR DESCRIPTION
**Pull request by mazurGit**

### CODE REVIEW

File: packages/mobile/.eslintrc.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ rules:
   '@typescript-eslint/quotes':
     - error
     - single
+    - "avoidEscape": true
```
Comment (AleksandrTermenzhy) [inline packages/mobile/.eslintrc.yml:51]:
```suggestion
    - avoidEscape: true
```

File: packages/frontend/package-lock.json [hunk 2]
```diff
@@ -49,10 +49,12 @@
       "name": "@vse-bude/shared",
       "version": "0.0.1",
       "dependencies": {
+        "joi": "^17.6.0",
```
Comment (AleksandrTermenzhy) [inline packages/frontend/package-lock.json:52]: These (and above) seem to be unrelated changes

File: packages/mobile/src/localization/languages/en.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    SING_IN: 'Sing in',
+    SING_UP: 'Sing up',
+    EMAIL: 'Email',
+    EMAIL_HINT: 'Enter your email',
+    PASSWORD: 'Password',
+    PASSWORD_HINT: 'Enter your password',
+    NAME: 'Name',
+    NAME_HINT: 'Enter your name',
+  },
+};
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/languages/en.js:12]: Line at the end

File: packages/mobile/src/localization/i18n.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import { ua, en } from './languages/languages';
+
+i18n
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/i18n.js:5]: The export should be kept within `localization.ts` file:
```
src/localiation
- localization.ts (here we export everything we need from this module)
- i18n/i18n.ts (here we keep i18n init)
- languages/languages.ts (existing structure is correct)
```

File: packages/mobile/src/index.tsx [hunk 5]
```diff
@@ -6,6 +6,7 @@ import { Navigation } from '~/navigation/navigation';
 import { store } from '~/store/store';
 import { NavigationDarkTheme, NavigationTheme } from '~/config/config';
 import { useColorScheme } from '~/hooks/hooks';
+import './localization/i18n';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/index.tsx:9]: Given that we don't use default exports I assume it should be something like this?

```suggestion
import { i18n } './localization/localization';
```

Comment (mazurGit) [reply packages/mobile/src/index.tsx:9]: This example was in documentation. Other way it doesn't work .

File: packages/mobile/src/localization/languages/languages.ts [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import { ua } from './ua';
+import { en } from './en';
+
+export { ua, en };
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/languages/languages.ts:4]:
```
export { ua } from './ua';
export { en } from './en';
```

File: packages/mobile/src/localization/localization.ts [hunk 7]
```diff
@@ -0,0 +1 @@
+export * from './i18n';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/localization.ts:1]:
```suggestion
export { i18 } from './i18n';
```

File: packages/mobile/src/index.tsx [hunk 8]
```diff
@@ -6,6 +6,7 @@ import { Navigation } from '~/navigation/navigation';
 import { store } from '~/store/store';
 import { NavigationDarkTheme, NavigationTheme } from '~/config/config';
 import { useColorScheme } from '~/hooks/hooks';
+import './localization/localization';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/index.tsx:9]:
```suggestion
import { i18 } from './localization/localization';
```

**Review by AleksandrTermenzhy — APPROVED** [review]

**Comment by AleksandrTermenzhy** [comment]
Please, follow PR-naming rules `VBM-{ticketNumber}: {ticket short description}`
